3.7/graphs-smart-graphs.md

```diff
@@ -71,6 +71,20 @@ overhead. The more subgraphs are touched the more network cost will apply.
 However the overall performance is never worse than the same query using a
 General Graph.
 
+Benefits of DisjointSmartGraphs
```

Comment:
What is the spelling? I assumed DisjointSmartGraphs

Reply:
DisjointSmartGraphs is correct.